From: Nicolas Saenz Julienne <nsaenzjulienne@suse.de>
Date: Thu, 2 Apr 2020 19:22:46 +0200
Subject: ARM: dts: Use upstream CMA configuration

Now that the kernel command line has precedence over the device tree,
we can use the upstream CMA setup without breaking backward
compatibility.

Signed-off-by: Nicolas Saenz Julienne <nsaenzjulienne@suse.de>
---
 arch/arm/boot/dts/bcm2711-rpi-4-b.dts | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
index 83a8431d9e5f..996112381eef 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
@@ -195,7 +195,7 @@
 
 / {
 	chosen {
-		bootargs = "coherent_pool=1M 8250.nr_uarts=1 cma=64M";
+		bootargs = "coherent_pool=1M 8250.nr_uarts=1";
 	};
 
 	aliases {
@@ -215,10 +215,6 @@
 	};
 
 	/delete-node/ wifi-pwrseq;
-
-	reserved-memory {
-		/delete-node/ linux,cma;
-	};
 };
 
 &mmcnr {
